specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/examples/WafListAllPolicies.json

```diff
@@ -10,7 +10,7 @@
           {
             "name": "Policy1",
             "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies/Policy1",
```

Comment:
Should this also be changed to camel-case in examples and specs both to keep consistency between specs? I.e. `ApplicationGatewayWebApplicationFirewallPolicies` -> `applicationGatewayWebApplicationFirewallPolicies`